app/views/cart/show.html.erb

```diff
@@ -31,6 +31,7 @@
       <td>
         <center>
           <p><%=quantity%></p>
+          <p><%= button_to "+", "/cart/#{item.id}",params: {'operation' => "add"}, method: :patch %></p>
```

Comment:
Greg showed me the hidden params and used it here. Pretty awesome!

Reply:
Nice work! I'll have to get schooled on that as well